.github/workflows/create-pullrequest-prerelease.yml

```diff
@@ -14,10 +14,10 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Use Node.js 16.13
+      - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 16.13
+          node-version-file: package.json
           cache: 'npm' # cache ~/.npm in case 'npm ci' needs to run
```

Comment:
None of the other steps include this. Should we include it?

Reply:
ah... I'd remove it and we can put these back in a separate PR 🙂

Reply:
removed 🙂